src/main/java/Graphics/AxesDrawer.java

```diff
@@ -27,13 +27,17 @@
       RGBA xColor = new RGBA("FFFF0000");
       RGBA yColor = new RGBA("FF0044FF");
 
-      // Y axis
-      for (int y = 0; y < h; y++) {
-        pixels[y * w + xOrigin] = yColor.blend(new RGBA(pixels[y * w + xOrigin]), 0.6f).toInt();
+      if ((xOrigin > 0) && (xOrigin < w)) {
```

Comment:
That's a better way than what I had. I did a bound on 0 <= ... <= pixels.length, but apparently that doesn't entirely solve the problem. Nice!